packages/mjml-core/src/helpers/skeleton.js

```diff
@@ -76,10 +76,9 @@ export default function skeleton(options) {
         }
         ${headRaw.filter(negate(isNil)).join('\n')}
       </head>
-      <body${
-        backgroundColor === ''
-          ? ''
-          : ` style="background-color:${backgroundColor};"`
+      <body${backgroundColor === ''
+        ? ' style="word-spacing:normal"'
```

Comment:
```
style="word-spacing:normal;${backgroundColor ? `background-color:${backgroundColor};` : ''}"
```

Maybe something like that to avoid repetition ?

Reply:
@iRyusa Thanks for the suggestion I've updated that now